cmd/junitreport/pkg/api/test_case.go

```diff
@@ -0,0 +1,38 @@
+package api
+
+import (
+	"strings"
+	"time"
+)
+
```

Comment:
Following origin conventions, I'd say package `api` should contain only types. The implementation should be placed somewhere else.